toolkit/tools/pkg/isomakerlib/isomaker.go

```diff
@@ -92,6 +92,7 @@ func NewIsoMaker(unattendedInstall bool, baseDirPath, buildDirPath, releaseVersi
 		fetchedRepoDirPath: isoRepoDirPath,
 		outputDirPath:      outputDir,
 		imageNameBase:      imageNameBase,
+		grubCfgPath:        isoGrubCfg,
```

Comment:
I thought the 64k ISO building was something we wanted to do through Prism. Did something change?